src/coprocessor/dag/rpn_expr/mod.rs

```diff
@@ -47,6 +50,12 @@ fn map_compare_int_sig<F: CmpOp>(
     })
 }
 
+/// Generate RPN functions from given ScalarFuncSig.
+///
+/// If the ScalarFuncSig means compare function for integer, function map_compare_int_sig will be called
```

Comment:
The implementation detail doesn't need to be explained in the function documentation